examples/ConfluentCloud/Program.cs

```diff
@@ -59,9 +59,9 @@ static void Main(string[] args)
                 SaslPassword = "<ccloud secret>"
             };
 
-            using (var producer = new Producer<Null, string>(pConfig))
+            using (var producer = new Producer(pConfig))
             {
-                producer.ProduceAsync("dotnet-test-topic", new Message<Null, string> { Key = null, Value = "test value" })
+                producer.ProduceAsync("dotnet-test-topic", new Message<Null, string> { Value = "test value" })

```

Comment:
this is also possible: `producer.ProduceAsync("dotnet-test-topic", new Message { Value = Encoding.UTF8.GetBytes("test value")`.

Reply:
this would not be strongly typed.